### DISCUSSION TITLE
Internal pdf viewer doesn't display \xcolor styling

### DISCUSSION THREAD

**Discussion by jobiewong** [James-Yu/LaTeX-Workshop#3332] [Q&A] [answered]
The internal pdf viewer has suddenly stopped displaying any of the colour stylings added using the \xcolour package (i.e. \color{red} Lorem ipsum... no longer changes to red).

This also applies to coloured paths in other pdfs that have been included as figures through \includegraphics. Lines or shapes created in Illustrator and saved as a pdf do not show up as coloured in the internal pdf viewer. However, images saved as PDFs do show in colour. 

This only broke in that last couple of days and no settings were changed in the extension to cause it.

### Example Latex syntax:
![Code_fBn3vfSkdr](https://user-images.githubusercontent.com/81576910/173184748-6784bd49-6d31-46eb-ba3e-3aace940e083.png)

### View in VScode
![Code_EJLKpi5XBP](https://user-images.githubusercontent.com/81576910/173184766-9a62a00e-3caa-4ea5-85a0-5762fc7853dd.png)

### View in Firefox
![firefox_li6otdyWMh](https://user-images.githubusercontent.com/81576910/173184778-a9aeff67-8630-4587-ba37-364346b49a74.png)

---
_Edit:_
I see someone else has reported having this issue.

For reference, this is a diagram created in Illustrator using paths and shapes, as well as an image. This was then saved as a pdf and imported using \includegraphics.

### View in VSCode internal viewer
![Code_yNEkaBXxWQ](https://user-images.githubusercontent.com/81576910/173186048-71989d96-d90d-40fa-a39c-db8d3c179616.png)
al viewer

As you can see, the image (the spectrogram) is still in colour, but the lines and paths are not. The correct view can be seen below.

### View in Firefox Browser viewer
![firefox_bPLElaLIYm](https://user-images.githubusercontent.com/81576910/173186064-a300b6e1-ca11-4a92-9727-98a72166a5cd.png)

**Comment by jobiewong** [comment, marked as answer]
This issue seems to have just fixed itself? I changed nothing and the colours are working again. They were broken for around three days.

**Reply by Hanbing59** [reply]
Hi James, the issue you just posted is quite similar to mine. But, not as in your case, my situation does not change. Yes, all figures inserted as PDFs are black-white as well as texts highlighted with different colors. Images inserted as .PNG shows as they should be.
